gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/ReplicationMetaData.java

```diff
@@ -0,0 +1,41 @@
+package gobblin.data.management.copy.replication;
+
+import com.google.common.base.Optional;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+/**
+ * Class used to represent the meta data of the replication
+ * @author mitu
+ *
+ */
+
+@AllArgsConstructor
+public class ReplicationMetaData {
```

Comment:
This class seems unnecessarily restrictive, in that it will be hard in the future to add new fields, is there a problem with simply using a map? Otherwise we should really think about what are the correct fields here.